crates/astria-proto/proto/astria/primitive/v1/types.proto

```diff
@@ -0,0 +1,8 @@
+syntax = "proto3";
+
+package astria.primitive.v1;
+
+message U128 {
```

Comment:
Can we comment here on how to interpret (even if somewhat apparent), since any other languages would need to add custom support?

Reply:
Hah, I forgot to push the comment. Sorry about that and I agree :-)